### DISCUSSION TITLE
Acces to log when failure

### DISCUSSION THREAD

**Discussion by pemile29** [cyberbotics/wrestling#29] [General]
After a failure the link given to the "action logs" contains no details of on the cause of the error ?

**Comment by omichel** [comment]
It seems was some problem on the server as several instances of Webots seem to run in parallel. I just fixed this. Please re-run your jobs and let me know here if you believe the problem persists.

The error was:
```
docker: Error response from daemon: driver failed programming external connectivity on endpoint intelligent_kowalevski (c48cf8cc4ada96f1004e0d6738af705794da1a60f0bac3632870cca6aee6ab6e): Bind for 0.0.0.0:3005 failed: port is already allocated.
```
Which is symptomatic of another instance of Webots running on port 3005 preventing any new instance of Webots to start...
We will try to improve the robustness of the server with respect to this.

**Reply by pemile29** [reply]
the problem seems to be solved, given the variety of languages supported, the error logs are supposed to contain the error traces of the python or java interpreters of the different participants?

**Reply by zicklag** [reply]
The logs contain the stderr and stout printed by the robot controllers for both participants.

So, regardless of the language, however you print out to the "terminal" will be visible in the logs.
